lib/index.js

```diff
@@ -192,27 +153,13 @@ export default {
           return [];
         }
 
-        // Construct the list of linters to be passed to the CLI by looking at
-        // the linters made available for our language grammar by the LinterMap,
-        // and whether or not the engines are enabled in a user's config file.
-        if (Object.prototype.hasOwnProperty.call(linterMap, grammarName) === true) {
-          // for (const linter of linterMap[grammarName]) {
-          linterMap[grammarName].forEach((linter) => {
-            linterNames.add(linter);
-          });
-        }
-        const configEnabledEngines = getEnabledEngines(configurationFilePath);
-        const linterEnabledEngines = Array.from(linterNames).filter(
-          linterName => configEnabledEngines.has(linterName));
-
         // Construct the command line invocation which runs the Code Climate CLI
         const relpath = atom.project.relativizePath(filePath).pop();
-        const execArgs = ['analyze', '-f', 'json'].concat(
-          getEngineArgs(linterEnabledEngines),
-          [relpath, '</dev/null']);
+        const execArgs = ['analyze', '-f', 'json', relpath];
         const execOpts = {
           cwd: dirname(configurationFilePath),
           uniqueKey: `linter-codeclimate::${relpath}`,
+          env: process.env,
```

Comment:
That's odd, it looks like the only cases where `null` would be returned are if `codeclimate` is crashing or if the process was killed due to the unique spawn constraint.

Since this is failing for you locally can you open the developer tools in the spec window and check the Console? If it's due to `codeclimate` crashing there should be more information printed there.